thirdeye/thirdeye-spi/src/main/java/org/apache/pinot/thirdeye/datalayer/pojo/AlertConfigBean.java

```diff
@@ -211,7 +211,7 @@ public String toString() {
     String properties;
 
     public String getType() {
-      if (StringUtil.isBlank(type)) {
+      if (isBlank(type)) {
```

Comment:
You can use StringUtils from the `commons.lang3`?

Reply:
Thanks. I'll replace it.